udata/core/dataservices/rdf.py

```diff
@@ -119,7 +121,13 @@ def dataservice_to_rdf(dataservice: Dataservice, graph=None):
     d.set(DCT.identifier, Literal(identifier))
     d.set(DCT.title, Literal(dataservice.title))
     d.set(DCT.description, Literal(dataservice.description))
-    d.set(DCT.issued, Literal(dataservice.created_at))
+
+    # created
+    set_harvested_date(dataservice, d, DCT.created, "created_at", fallback=dataservice.created_at)
+    # issued
+    set_harvested_date(dataservice, d, DCT.issued, "issued_at", fallback=dataservice.created_at)
+    # modified
+    d.set(DCT.modified, Literal(dataservice.metadata_modified_at))
```

Comment:
Is it possible to copy/paste the PR explanation about why no `modified` fallback vs created / issued?